modules/Microsoft.DocumentDB/databaseAccounts/sqlDatabases/containers/deploy.bicep

```diff
@@ -7,9 +7,24 @@ param sqlDatabaseName string
 @description('Required. Name of the container.')
 param name string
 
+@description('Optional. Indicates how long data should be retained in the analytical store, for a container. Analytical store is enabled when ATTL is set with a value other than 0. If the value is set to -1, the analytical store retains all historical data, irrespective of the retention of the data in the transactional store.')
+param analyticalStorageTtl int = 0
+
+@description('Optional. The conflict resolution policy for the container. Conflicts and conflict resolution policies are applicable if the Azure Cosmos DB account is configured with multiple write regions.')
+param conflictResolutionPolicy object = {}
+
+@maxValue(2147483647)
+@minValue(-1)
+@description('Optional. Default time to live (in seconds). With Time to Live or TTL, Azure Cosmos DB provides the ability to delete items automatically from a container after a certain time period. If the value is set to "-1", it is equal to infinity, and items dont expire by default.')
+param defaultTtl int = -1
+
 @description('Optional. Request Units per second.')
 param throughput int = 400
 
+@minValue(0)
+@description('Optional. Specifies the Autoscale settings and represents maximum throughput, the resource can scale up to. If value is set to 0, then the property will be set to null.')
+param autoscaleSettingsMaxThroughput int = 0
```

Comment:
Same question here

Reply:
Good call I have aligned to our implementation and provided additional guidance in the parameter